mediasoup-client-ios/src/C++Wrapper/TransportWrapper.mm

```diff
@@ -110,7 +110,6 @@ +(void)nativeClose:(NSValue *)nativeTransport {
     MSC_TRACE();
     
     [TransportWrapper extractNativeTransport:nativeTransport]->Close();
-    [nativeTransport release];
```

Comment:
hi @fedulvtubudul , I'm curious why is this not necessary.. when/where is it released instead?
not an expert btw but i've been using this library and faced some issues.
thanks!

Reply:
This fragment looks fragile. If user does not close the transport, the transport object will leak. If user will try to close the transport twice, app will crash while accessing already deallocated object.

With this change, app does not crash after leaving a channel (or ending a call). BUT yes, it has memory leaks, lots of them actually. I'm trying to solve the leaks, you can check the progress here: https://github.com/VLprojects/mediasoup-ios-client/commit/bb2295c8e3dec47b69cc02070b4307a4ce82c719, but it's not finished yet.